aiven/client/pretty.py

```diff
@@ -62,69 +89,127 @@ def format_item(key, value):
     return formatted
 
 
+class PreparedTableData:
+    """PreparedTableData represents prepared data for methods which render tables."""
+
+    def __init__(self, fields, table_rows, horizontal_fields, vertical_fields):
+        self._fields = fields
+        self._table_rows = table_rows
+        self._horizontal_fields = horizontal_fields
+        self._vertical_fields = vertical_fields
+
+    @property
```

Comment:
This could be a simple class to hold the values, the use of @property doesn't seem to add any benefit from accessing the attributes directly.

Reply:
It's a little bit of future proofing: all the table routines need 4 basic bits of data - a list of fields, a list of rows, and lists of which fields are horizontal and vertical.

Currently this calculation happens in `prepare_table` for want of a clear idea of where it's going. Possibly the object in the future on initialization should instead just receive the data and set that up internally.

Reply:
If at some later date it needs to be modified it is easy to do without breaking any of the code that depends on it. As it stands currently there doesn't seem to be any reason to use a property.

Reply:
Fixed.